app/views/bgeigie_imports/index.html.erb

```diff
@@ -51,6 +51,17 @@
 <%- if @bgeigie_imports.empty? -%>
   <%= t("none_found") -%>
 <%- else -%>
+
+  <ul class="inline pull-right">
+    <%-if user_signed_in? %>
+      <%- if current_user.moderator? -%>
```

Comment:
You might be able to simplify these two `if` statements with just one `if` statement like `if current_user.try!(:moderator?)`. 

`current_user` would be `nil` if user is not logged-in. So, `current_user.try!(:moderator?)` would be `nil` which is falsy value. If use is logged-in, then `current_user` would be a `User` object, and `current_user.try!(:moderator?)` would return value depending on user's status.

Reply:
Another place a view helper would be nice `is_moderator?` could be a good name.